hedera-node/src/main/java/com/hedera/services/txns/token/TokenMintTransitionLogic.java

```diff
@@ -100,6 +105,13 @@ public void doStateTransition() {
 		accountStore.persistAccount(token.getTreasury());
 	}
 
+	private void validateMinting(Token token, TokenMintTransactionBody op) {
+		if (token.getType() == NON_FUNGIBLE_UNIQUE) {
+			final var proposedTotal = tokenStore.currentMintedNfts() + op.getMetadataCount();
+			validateTrue(validator.isPermissibleTotalNfts(proposedTotal), MAX_NFTS_IN_PRICE_REGIME_HAVE_BEEN_MINTED);
```

Comment:
nit: based on the context, `MAX_NFTS_IN_PRICE_REGIME_HAVE_BEEN_MINTED` is a little vague here. Maybe `MORE_THAN_MAX_NFTS_IN_PRICE_REGIME_HAVE_BEEN_MINTED` or something like that will be more explicit to tell this is an error condition.

Reply:
No, it means max has been reached and we cannot mint any more. Saying more than max nfts have been minted is actually incorrect.

Reply:
Got it. My understanding was wrong. Thx for the clarification.